src/serialization/src/main/kotlin/org/vaccineimpact/api/serialization/MontaguSerializer.kt

```diff
@@ -122,10 +124,18 @@ open class MontaguSerializer : Serializer
 
     override val serializeNullsTo = "<NA>"
 
+    private val decimalFormat = DecimalFormat("###.##")
+
+    private fun serializeBigDecimal(value: BigDecimal) : String
+    {
+        return decimalFormat.format(value)
+    }
+
     override fun serializeValueForCSV(value: Any?) = when (value)
     {
         null -> serializeNullsTo
         is Enum<*> -> serializeEnum(value)
```

Comment:
Is checking for BigDecimal the right thing to do here, or should generalize to other floating point types we might see?